src/ai/provider.ts

```
@@ -92,6 +100,54 @@ export const createModel = ({
+          case ModelProvider.OPENAI:
+            if (!process.env.OPENAI_API_KEY) {
+              return {
+                success: false,
+                message: "OPENAI_API_KEY is not set",
+                object: null,
+              };
+            }
+            const openai = createOpenAI({
+              apiKey: process.env.OPENAI_API_KEY,
+              compatibility: "strict",
+            });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)